tools/ofono-setup

```diff
@@ -85,6 +85,8 @@ MODEM_COUNT=0
 # Those accounts are created from there
 if [ "$(getprop ro.build.version.sdk 99)" -ge "27" ]; then
     echo "Android SDK version 27 or greater detected. No modem detection will be made."
+    # Remove an eventual account that got here before the fix
+    mc-tool remove ofono/ofono/account0
```

Comment:
for account in $EXISTING_OFONO_ACCOUNTS; do
        if [ -z "${account##*account*}" ]; then
           mc-tool remove "$account"
        fi
    done